### PR TITLE
refactor: clean up eco_challenge experiment and layout

### PR DESCRIPTION
**Pull request by eddieferrer**

### CODE REVIEW

File: src/components/LoanCollections/KivaClassicSingleCategoryCarousel.vue [hunk 1]
```diff
@@ -30,9 +22,7 @@
 </template>
 
 <script>
-import IconClimateChallenge from '@/assets/icons/inline/eco-challenge/globe-leaf.svg';
 import KivaClassicLoanCarousel from '@/components/LoanCollections/KivaClassicLoanCarousel';
-import KvPill from '@/components/Kv/KvPill';
```
Comment (dyersituations) [inline src/components/LoanCollections/KivaClassicSingleCategoryCarousel.vue:35]: It looks like we don't use this component elsewhere, safe to delete the component entirely?

File: src/pages/Lend/LoanChannelCategoryClimateExperiment.vue [hunk 2]
```diff
@@ -63,7 +63,6 @@
 				<kiva-classic-single-category-carousel
```
Comment (dyersituations) [inline src/pages/Lend/LoanChannelCategoryClimateExperiment.vue:63]: Are we saving this component?

File: src/components/BorrowerProfile/LendCta.vue [hunk 3]
```diff
@@ -313,12 +313,6 @@
 					</div>
 				</kv-grid>
 			</transition>
-
-			<eco-challenge-lightbox
```
Comment (dyersituations) [inline src/components/BorrowerProfile/LendCta.vue:317]: Are we saving this component?

File: src/components/BorrowerProfile/LendCta.vue [hunk 4]
```diff
@@ -341,15 +335,9 @@ import {
 	isBetween25And500
 } from '@/util/loanUtils';
 import { createIntersectionObserver } from '@/util/observerUtils';
-import {
-	getExperimentSettingCached,
-	trackExperimentVersion
-} from '@/util/experimentUtils';
-import { achievementsQuery, hasMadeAchievementsProgression } from '@/util/ecoChallengeUtils';
```
Comment (dyersituations) [inline src/components/BorrowerProfile/LendCta.vue:348]: Same with these utils?

File: src/components/LoanCollections/KivaClassicSingleCategoryCarousel.vue (hunk 1 above)
Comment (mcstover) [reply src/components/LoanCollections/KivaClassicSingleCategoryCarousel.vue:35]: I'd like to keep these around for now as I think we'll prob revisit when we fill out our new component lib.

**Review by mcstover — APPROVED** [review]
I know @eddieferrer wanted to keep around some of the utils and templates as they may be needed later. Lots of nice cleanup and simplification otherwise!

File: src/components/LoanCollections/KivaClassicSingleCategoryCarousel.vue (hunk 1 above)
Comment (eddieferrer) [reply src/components/LoanCollections/KivaClassicSingleCategoryCarousel.vue:35]: Yeah these are actually quite nice looking.

**Comment by eddieferrer** [comment]
@dyersituations It is unclear to me wether we will have another challenge in the future, so I left the components in here, since there is a lot of work put in those, and it would certainly be easier to repurpose them for a future challenge. I will add comments to them to make this clear

**Review by dyersituations — APPROVED** [review]

**Pull request merged**